src/molecule/driver/delegated.py

```
@@ -128,9 +135,132 @@ def login_options(self, instance_name: str) -> dict[str, str]:
+        Args:
+            extra_args: Additional ansible-inventory CLI arguments, e.g.
+                ``["--host", name]`` or ``["--list"]``.
+
+        Returns:
+            The parsed JSON output, or None if the command failed for any
+            reason (ansible-inventory not available, no matching host,
+            malformed output, etc).
+        """
+        cmd = ["ansible-inventory", *extra_args, *self._ansible_inventory_args()]
+        try:
+            result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)